layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -863,7 +863,11 @@ ones created by `magit' and `dired'."
 (defun spacemacs/copy-file-path ()
   "Copy and show the file path of the current buffer."
   (interactive)
-  (if-let* ((file-path (spacemacs--file-path)))
+  (if-let* ((file-path (or (spacemacs--file-path)
+                           (and (eq major-mode 'dired-mode)
```

Comment:
Prefer to use `derived-mode-p` over `(eq major-mode ...)`

Reply:
Changed.